### PR TITLE
Add descriptions for "page::*" and "user::*"

### PR DESCRIPTION
**Pull request by lionel-m**
See issue #389

### CODE REVIEW

**Comment by lionel-m** [comment]
@aschempp and @Aybee : Does it solve the ticket #389 ?

**Comment by lionel-m** [comment]
I need the German translation for `{{user::*}}` please

**Comment by Aybee** [comment]
Yes this will solve ticket #389, thank you. But I thnk it had to be table `tl_member`. Am I right?

**EN**

```
+  <td><code>{{user::*}}</code></td>
+  <td>This tag will be replaced with the content of a field in the table `tl_member`
+      in relation to the current member. (replace * with the name of the field).</td>
+</tr>
+<tr>
```

**DE**

```
+  <td><code>{{user::*}}</code></td>
+  <td>Dieses Tag wird mit dem Inhalt eines Feldes von `tl_member` des angemeldeten
+      Mitglieds ersetzt (ersetzen Sie * mit dem Feldnamen).</td>
+</tr>
+<tr>
``
```

**Comment by lionel-m** [comment]
You are right and thx for the translation.

We can also change the term "User" with "Front end user" it would be more understandable...

> Front end user properties
> 
> The following insert tags allow you to display any property of the currently logged in front end user.

etc.

WDYT?

**Comment by Aybee** [comment]
Maybe we should use "member" instead of "front end user" as "member" is the term in back end.

I think in german translation we also should change "angemeldeten Benutzers", which means "logged in user" to "angemeldeten Mitglieds", which means "logged in member" for whole text regarding `{{user::xxx}}`.

Also the headline "Benutzereigenschaften" should be changed to "Mitgliedereingenschaften" as in Contao we have Members (Mitglieder) and Users (Benutzer) in the Accont manager (Benutzerverwaltung) in back end.

**Comment by lionel-m** [comment]
@Aybee I created a new PR for that: #436.

I agree but I added the term "front end user" once in the paragraph to avoid a confusion after with the prefix `{{user::`...

**Comment by Aybee** [comment]
Looks good to me. Thank you 👍

File: manual/en/04-managing-content/insert-tags.md [hunk 1]
```diff
@@ -165,6 +165,11 @@ logged in user.
   <th>Description</th>
 </tr>
 <tr>
+  <td><code>{{user::*}}</code></td>
+  <td>This tag will be replaced with the content of a field in the table `tl_member`
+      in relation to the current user. (replace * with the name of the field).</td>
```
Comment (aschempp) [inline manual/en/04-managing-content/insert-tags.md]: this should be *current member*, right?

**Review by aschempp — CHANGES_REQUESTED** [review]

File: manual/en/04-managing-content/insert-tags.md (hunk 1 above)
Comment (Aybee) [reply manual/en/04-managing-content/insert-tags.md]: Yes, as I showed here earlier https://github.com/contao/docs/pull/436#issuecomment-261413402.

File: manual/fr/04-gestion-du-contenu/balises-insertion.md [hunk 2]
```diff
@@ -166,6 +166,11 @@ propriété du membre actuellement connecté.
   <th>Description</th>
 </tr>
 <tr>
+  <td><code>{{user::*}}</code></td>
+  <td>Cette balise sera remplacée par le contenu d'un champ de la table `tl_member`
+      par rapport à l'utilisateur courant. (remplacez * par le nom du champ).</td>
```
Comment (aschempp) [inline manual/fr/04-gestion-du-contenu/balises-insertion.md]: @lionel-m does `l'utilisateur` equal *member* or is this still *user*?

Comment (lionel-m) [reply manual/fr/04-gestion-du-contenu/balises-insertion.md]: You right it is member

> par rapport au membre courant.

**Comment by aschempp** [comment]
Thanks a lot!

**Comment by lionel-m** [comment]
@aschempp #436 and #437 are not in the 4.2 branch and master.

**Comment by aschempp** [comment]
Done 😊

**Comment by lionel-m** [comment]
👍

**Pull request merged**